test.py

```diff
@@ -75,6 +75,15 @@
     librariespath = os.path.normpath(os.path.join(os.environ.get('APPDATA'), '.openmodelica', 'libraries'))
   else:
     librariespath = os.path.normpath(os.path.join(os.environ.get('HOME'), '.openmodelica', 'libraries'))
+
+# add openmodelica libraries path if the Modelica libraries are not found in the libraries path
+modelicaLibpath = ''
+if len(glob.glob('Modelica*', root_dir=librariespath)) == 0:
+  if isWin:
+    modelicaLibpath = ';' + os.path.normpath(os.path.join(os.environ.get('APPDATA'), '.openmodelica', 'libraries')).replace('\\','/')
+  else:
+    modelicaLibpath = ':' + os.path.normpath(os.path.join(os.environ.get('APPDATA'), '.openmodelica', 'libraries')).replace('\\','/')
```

Comment:
Are you sure `APPDATA` is defined on non-Windows machines?

Reply:
Sorry, you did not observe DRY/duplicated code, so the same thing happens in another place in the PR, hence my remark "Same below": https://github.com/OpenModelica/OpenModelicaLibraryTesting/pull/122/files#diff-f7c50a89e57f8fd5080d06d65e93480d1c23010919e4c42d72563090c7865414R37, which you did not update just now AFAICT.

I'm actually confused how this worked correctly in the first place -- is this tested by CI, or you locally on a Linux and Windows machine?

Reply:
Hi Cristopher, you are right... I forgot that the same code has been added also to the `testmodel.py` script. I'll update the PR.

Unfortunately I've not a linux machine to test the script locally in this moment (I've to build a VM to do this ... it is in my stack...) so I test on WIN 11 only. For the test under linux I use the CI (I understood that all the PR are tested by the CI under Linux...).

Reply:
Yes, PR are tested but not with the full functionality (no sync, etc), see here what is ran for the test:
https://github.com/OpenModelica/OpenModelicaLibraryTesting/blob/master/.github/workflows/test.yml

However, usually we detect problems really quickly because the LibraryTest jobs will start failing :)